cmd/cloudFoundryDeploy.go

```diff
@@ -226,44 +218,22 @@ func handleMTADeployment(config *cloudFoundryDeployOptions, command command.Exec
 }
 
 type deployConfig struct {
-	DeployCommand   string
-	DeployOptions   []string
-	AppName         string
-	ManifestFile    string
-	SmokeTestScript []string
+	DeployCommand string
+	DeployOptions []string
+	AppName       string
+	ManifestFile  string
 }
 
 func handleCFNativeDeployment(config *cloudFoundryDeployOptions, command command.ExecRunner) error {
 
-	deployType, err := checkAndUpdateDeployTypeForNotSupportedManifest(config)
-
-	if err != nil {
-		return err
-	}
-
 	var deployCommand string
-	var smokeTestScript []string
```

Comment:
From that I derive that there is no smoke-test for cf native deployment. Wouldn't it be reasonable to have that also here?

Reply:
Earlier smoke tests were offered by blue-green deployment plugin which was used for cf native build tools. Since, we want to go away from usage of that plugin and hence eventually the smoke tests offered by it are gone now. Further information can be seen [here](https://github.com/bluemixgaragelondon/cf-blue-green-deploy?tab=readme-ov-file#overview). Still any remarks?